packages/constants/src/constants.ts

```diff
@@ -66,7 +66,9 @@ export type TAppNotificationOfKind<T extends TAppNotificationOfDomain> =
   };
 export type TAppNotificationApiError<ExtraFields extends {} = {}> = {
   message: string;
-  code?: string;
+  extensions?: {
+    code?: string;
+  };
```

Comment:
should we optionally supply `code?: string` for backwards compatibility, and handle it downstream in the function with something like `const extensionErrorCode = props.error.code || props.error.extensions?.code;`?

Reply:
Yes we should support both for backwards compatibility. Can you update the PR with that change? Thanks!

Reply:
I have some time now, I can do that.

Reply:
Thank you!